microsoft-edge/devtools-guide-chromium/experimental-features/focus-mode.md

```diff
@@ -0,0 +1,180 @@
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 03/15/2022
+---
+# Reduce the complexity of DevTools with Focus Mode
+
+DevTools has over 30 different tools and many toolbar icons. The current DevTools user interface consists of:
+
+* A main toolbar containing icons such as the **Inspect** and **Device Emulation** tools, and a set of tabbed tools including the **Elements** and **Sources** tools.
+
+* A drawer toolbar containing more tabbed tools and icons.
+
+The overall experience can be visually and conceptually overwhelming.
+
+Focus Mode is a new experimental user interface for DevTools. It is designed to lower the learning curve for new users and reduce the complexity of DevTools without compromising its feature set.
+
+Focus Mode replaces the main toolbar with an **Activity Bar**, which is a compact horizontal or vertical toolbar. The **Activity Bar** contains small icons and corresponding tooltips. The **Activity Bar** makes it possible to open tools, change user settings, and access additional features from one location.
+
+Focus Mode also provides a **Quick View** list to open a second tool alongside the tool already selected in the **Activity Bar**.
+
+![Focus Mode, including the Activity Bar.](media/focus-mode.png)
+
+Focus Mode does not change how the tools work and what they look like. Focus Mode only changes the way tools and other customization features of DevTools can be accessed.
+
+
+<!-- ====================================================================== -->
+## Enable Focus Mode
+
+The Microsoft Edge team is currently experimenting with Focus Mode. Some users already have access to the feature while others need to enable it first.
+
+To check if you have Focus Mode enabled already, or to enable or disable it:
+
+1. Press `F1` to open **Settings**.
+
+1. Click **Experiments** and scroll down to the **Focus Mode** checkbox.
+
+   If the checkbox is selected, Focus Mode is already enabled. Select or clear it to enable or disable Focus Mode.
+
+    ![Screenshot of the Settings page in DevTools, showing the Experiments tab, scrolled down to the Focus Mode checkbox](media/focus-mode-pref.png)
+
+1. Close the **Settings** page. If you changed the Focus Mode experiment state, click the **Reload DevTools** button for the change to take effect.
+
+
+<!-- ====================================================================== -->
+## Main differences from the default user interface
+
+Here are the main differences between the DevTools default user interface and Focus Mode.
+
+### Activity Bar
+
+  The **Activity Bar** is the most visible difference from the default user interface. Initially, it appears as a vertical sidebar to the left of the DevTools window. The **Activity Bar** contains icons used to open tools and access help and settings.
```

Comment:
Pedantic nit, but I believe the default behavior is "adapt to dock location", so depending on where/if devtools are docked it may initially be a horizontal bar

Reply:
Good point, fixing